entries/w/withings.com.json

```diff
@@ -2,10 +2,12 @@
   "Withings": {
     "domain": "withings.com",
     "url": "https://www.withings.com/",
```

Comment:
As you're already editing the Withings entry it would be great if you could remove the url-element as we try to phase it out 🙂
```suggestion
```